fmn/fmn_fasshim.py

```diff
@@ -57,79 +74,96 @@ def make_fas_cache(**config):
             socket.setdefaulttimeout(timeout)
 
         log.info("Caching necessary user data")
-        for user in request['people']:
-            nick = user['ircnick']
-            if nick:
-                _cache.set(str(nick), user['username'])
-
-            email = user['email']
-            if email:
-                _cache.set(str(email), user['username'])
-
+        _add_to_cache(request['people'])
```

Comment:
I don't think you should do that replacement here, `_add_to_cache()` does not know how to handle the old fas2 format anymore.

Reply:
Same as before, missed this. Thanks.